lib/core/src/Cardano/Wallet/Api/Client.hs

```diff
@@ -317,6 +317,7 @@ addressClient =
     let
         _listAddresses
             :<|> _inspectAddress
+            :<|> _postScriptAddress
```

Comment:
small note: the endpoint suggest that we can post _any_ address here?

Reply:
yes, I "coined this name" due to the fact that ANY address could be constructed: base, reward account, enterprise, each with any credential. Of course there is also pointer address which was omitted in this PR (but could be added in the forthcoming)